ansible/roles/girder/tasks/main.yml

```diff
@@ -101,6 +105,15 @@
     state: present
   sudo: yes
 
+
+- name: Create directory for build
+  file: state=directory path="{{ root_dir }}/girder/clients/web/static/built"
+
+- name: Download fontello.zip
+  command: wget -O fontello.zip https://data.kitware.com/api/v1/file/57c5d1fc8d777f10f269dece/download
```

Comment:
It turns out that OpenJPEG 2.1.1 and later use libtiff 4.0.6, which breaks pylibtiff 0.4.0.  I've got a tested update to work with these in large_image (https://github.com/DigitalSlideArchive/large_image/pull/116).  Either that should be merged in and this script should explicitly call for libtiff 4.0.6, or we should back this off to OpenJPEG 2.1.